src/fuselage/CCPACSFuselageProfile.cpp

```diff
@@ -219,6 +221,51 @@ void CCPACSFuselageProfile::ReadCPACS(TixiDocumentHandle tixiHandle)
     Update();
 }
 
+// Write fuselage profile file
+void CCPACSFuselageProfile::WriteCPACS(TixiDocumentHandle tixiHandle, const std::string& profileXPath)
+{
+    // Set attribute "uID"
+    TixiSaveExt::TixiSaveTextAttribute(tixiHandle, profileXPath.c_str(), "uID", uid.c_str());
+    
+    // Set element "name"
+    TixiSaveExt::TixiSaveTextElement(tixiHandle, profileXPath.c_str(), "name", name.c_str());
+    
+    // Set element "name"
+    TixiSaveExt::TixiSaveTextElement(tixiHandle, profileXPath.c_str(), "description", description.c_str());
+    
+    // Set the element "point"
+    TixiSaveExt::TixiSaveElement(tixiHandle, profileXPath.c_str(), "pointList");
+
+    // TODO: symmetry!!!
+
+    //sprintf(profileXPath.c_str(), "%s/pointList", profileXPath.c_str());
+    // TODO : find how to get the number of 'point' (and eventually the definition...)
+    int pointCount = 1;
+    std::string elementPath = profileXPath + "/pointList";
+    for (int i = 1; i <= pointCount; i++) {
+        // TODO : multiple points is not managed yet
+        TixiSaveExt::TixiSaveElement(tixiHandle, elementPath.c_str(), "point");
```

Comment:
I don't really understand the code. Why do you create an empty point element? Why do you loop over pointCount, which is always 1? I think, this code should be tidied up a bit, or at least commented (why the code looks like now).